src/components/Personalization/index.js

```diff
@@ -140,6 +140,9 @@ const createPersonalization = ({ config, logger, eventManager }) => {
         showContainers();
 
         storeDecisions(decisionsStorage, decisions);
+
+        // eslint-disable-next-line consistent-return
```

Comment:
Rather than disabling the linting, change your code at the top of `onResponse` to this:

```
if (authoringModeEnabled) {
  return undefined;
}
```

Whether the team wants to continue with this `consistent-return` lint rule or not is something I'll try to bring up at a tech checkin.

Reply:
ok, thank you.